# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/rose_tree/LICENSE-MIT
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/rose_tree/LICENSE-MIT b/third_party/rust/khronos-egl/LICENSE-MIT
copy from third_party/rust/rose_tree/LICENSE-MIT
copy to third_party/rust/khronos-egl/LICENSE-MIT
--- a/third_party/rust/rose_tree/LICENSE-MIT
+++ b/third_party/rust/khronos-egl/LICENSE-MIT
@@ -1,10 +1,8 @@
-Copyright (c) 2015
-
 Permission is hereby granted, free of charge, to any
 person obtaining a copy of this software and associated
 documentation files (the "Software"), to deal in the
 Software without restriction, including without
 limitation the rights to use, copy, modify, merge,
 publish, distribute, sublicense, and/or sell copies of
 the Software, and to permit persons to whom the Software
 is furnished to do so, subject to the following